src/layouts/docs-view-layout/index.tsx

```diff
@@ -3,19 +3,25 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-import SidebarSidecarWithToc, {
-	SidebarSidecarWithTocProps,
-} from 'layouts/sidebar-sidecar-with-toc'
+import { OutlineNavWithActive } from 'components/outline-nav/components'
+import { OutlineLinkItem } from 'components/outline-nav/types'
+import SidebarSidecarLayout, {
+	SidebarSidecarLayoutProps,
+} from 'layouts/sidebar-sidecar'
 import { DocsVersionAlertBanner } from './components'
 
 /**
  * Lightweight wrapper around SidebarSidecarWithToc which passes along some docs
  * specific props.
  */
-const DocsViewLayout = (props: SidebarSidecarWithTocProps) => {
+const DocsViewLayout = ({
```

Comment:
All views that use `DocsViewLayout` must now ensure they're passing already-processed `outlineItems` from `getStaticProps`, not `headings` like before.